[PID/(Q)EAA Issuance - Detailed Flow] Discovery

SaraConsoliACN opened the pull request:
This commit aims to resolve the issue https://github.com/italia/eudi-wallet-it-docs/issues/263 adding the "trust-evaluation-mechanism" section reference  in chapter Detailed Flow - Step 1-4.

SaraConsoliACN committed:
[PID/(Q)EAA Issuance - Detailed Flow] Discovery

This commit aims to resolve the issue https://github.com/italia/eudi-wallet-it-docs/issues/263
adding the "trust-evaluation-mechanism" section reference  in chapter Detailed Flow - Step 1-4.

grausof commented on docs/en/pid-eaa-issuance.rst:
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
+`Trust Evaluation Mechanism <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/trust.html#trust-evaluation-mechanism>`_ section of this specification.
```
what do you think about using a relative path?

peppelinux replied:
I did it @grausof ^

peppelinux commented on docs/en/pid-eaa-issuance.rst:
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
+`Trust Evaluation Mechanism <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/trust.html#trust-evaluation-mechanism>`_ section of this specification.
```
```suggestion
`Trust Model section`_ Trust Evaluation Mechanism, of this specification.
```

peppelinux replied:
@grausof wdyt^

SaraConsoliACN committed:
Commit suggestion applied

Co-authored-by: Giuseppe De Marco <giuseppe.demarco@teamdigitale.governo.it>

SaraConsoliACN committed:
Editorial change

This additional commit aims to resolve the comment https://github.com/italia/eudi-wallet-it-docs/issues/263#issuecomment-2154523161

peppelinux commented on docs/en/pid-eaa-issuance.rst:
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
 
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
```
```suggestion
**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
```

peppelinux commented on docs/en/pid-eaa-issuance.rst:
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
 
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
+`Trust Model section`_ Trust Evaluation Mechanism, of this specification.
```
```suggestion
**Steps 1-4 (Discovery):** The User, using the Wallet Instance, selects the PID/(Q)EAA Provider from those listed in the list of trustworthy entities. The Wallet Instance then processes the Metadata for the selected PID/(Q)EAA Provider as defined in the `Trust Model section`_ Trust Evaluation Mechanism of this specification.
```

Commit:
Apply suggestions from code review

Commit:
Update docs/en/pid-eaa-issuance.rst

SaraConsoliACN committed:
hyperlink fixed

Commit:
Merge branch 'versione-corrente' into SaraConsoliACN-issue263
